src/js/components/Page.js

```diff
@@ -124,7 +124,7 @@ var Page = React.createClass({
       "page-body-content pod pod-tall flex",
       "flex-direction-top-to-bottom flex-item-grow-1",
       {
-        "flex-item-shrink-1": dontScroll
+        "flex-item-shrink-1 flush-bottom": dontScroll
```

Comment:
Can we please do this for Nodes page only?

Reply:
I will try to figure out a way to this only for the nodes page.

Reply:
I guess we only want this for the `NodesTable` page.